webapp/src/DataTransferObject/Shadowing/ContestEvent.php

```diff
@@ -6,10 +6,10 @@ class ContestEvent implements EventData
 {
     public function __construct(
         public readonly string $id,
-        public readonly string $name,
+        public readonly ?string $name,
```

Comment:
Although this is needed with the current WF repo, I'm not sure if we actually want to do this?

There will hopefully be a fix in the CDS and otherwise we can fix the config in the repo.

Reply:
Created another PR for this specific case.